recipes/hpcflow/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "hpcflow" %}
+{% set version = "0.2.0a50" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Also, this release doesn't seem to be on PyPI:

![Screenshot from 2023-06-21 08-16-18](https://github.com/conda-forge/staged-recipes/assets/950575/dd60ac4a-2af1-47fa-822c-2c6741836e91)

Reply:
Thanks for the review. We are currently refactoring the hpcflow project, and are transitioning from the package you found on PyPI (which will soon be archived) to this one: https://pypi.org/project/hpcflow-new2/

As you can see, the release linked in the recipe is available there.

As for the pre-release label, as you can see in the PyPI link above, we have never had a non-beta/alpha release, so I believe we fall under the [exception here](https://github.com/conda-forge/cfep/blob/main/cfep-05.md#exceptions).

Reply:
If you are, and/or know, the upstream dev could you request a stable release? Black was an exception b/c when installing with pip one did not need to use the `--pre` flag. (They finally dropped that exception BTW.)